samples/tfm/tfm_hello_world/sample.yaml

```diff
@@ -21,6 +21,7 @@ common:
         - "Hashing 'Hello World! .*'"
         - "SHA256 digest:"
         - "0x[0-9a-f]{64}"
+        - "Configuring MCU selection for LFXO"
```

Comment:
Please update the `README.rst` with this as well (maybe we should add an `literalinclude` to stop having to sync these manually)